pkg/azure/ocpgwdeployer.go

```diff
@@ -318,7 +312,7 @@ func (d *ocpGatewayDeployer) getAvailabilityZones(gwNodes []v1.Node) (stringset.
 	}
 
 	pager := resourceSKUClient.NewListPager(&armcompute.ResourceSKUsClientListOptions{
-		Filter: pointer.String(d.azure.Region),
+		Filter: to.StringPtr(d.azure.Region),
```

Comment:
go-autorest is deprecated and will no longer be supported soon, please don’t change this.

Reply:
Done. Sorry, it was not intentional.